src/cmd.c

```diff
@@ -1089,6 +1089,29 @@ static void pctx(atci_param_t *param)
     request_confirmation = true;
 }
 
+static void cw(atci_param_t *param)
+{
+    uint32_t freq, timeout;
+    int32_t power;
+
+    if (!atci_param_get_uint(param, &freq)) abort(ERR_PARAM);
+    if (!atci_param_is_comma(param)) abort(ERR_PARAM);
+    if (!atci_param_get_int(param, &power)) abort(ERR_PARAM);
+    if (!atci_param_is_comma(param)) abort(ERR_PARAM);
+    if (!atci_param_get_uint(param, &timeout)) abort(ERR_PARAM);
+
+    atci_printf("$CW: freq=%ld power=%ld timeout=%ld\r\n",  freq, power, timeout);
```

Comment:
Should we be sending this via the ATCI? This might throw off implementations that expect just `+OK` or `+ERR` response. Besides, I think it adds little value beyond checking that the number parsers worked correctly.

Alternatively, if you want to see that all the parameters have the right value, you could print the string to the debugging console with `log_debug`.

Reply:
Right, this was initially just to check the new int ATCI parser, but makes sense to keep it and move to log console.